vortex-array/src/canonical.rs

```diff
@@ -43,26 +42,23 @@ use crate::{Array, ArrayDType, IntoArray, ToArray};
 /// decompress it later to pass to a compute kernel, there are multiple suitable Arrow array
 /// variants to hold the data.
 ///
-/// To disambiguate, we choose a canonical physical encoding for every Vortex [`DType`], which
+/// To disambiguate, we choose a canonical physical encoding for every Vortex `DType`, which
```

Comment:
nit - removing the `[]` removes the link in rendered rustdoc.

Reply:
I think at some point I had deleted the `DType` import which is why I killed the link but it's back now, will change